### ISSUE TITLE
LSTM & RNN NLP 코드 2 (PyTorch, Classification)

### ISSUE THREAD

**Issue by sallyy1** [sallyy1/NLU-NLP#12]
# 3
### - Embedding Layer로 Vocab Size => Word Vec Size(보통 'embedding_dim'이라고 부르는 변수) 차원 축소 후, Word Vec Size를 LSTM의 input size로 입력

- 김기현님 예제 (https://github.com/kh-kim/simple-ntc/blob/master/simple_ntc/models/rnn.py)
- 구조
        self.input_size = input_size  # vocabulary_size
        self.word_vec_size = word_vec_size
        self.hidden_size = hidden_size
        self.n_classes = n_classes
        self.n_layers = n_layers
        self.dropout_p = dropout_p

        super().__init__()

        self.emb = nn.Embedding(input_size, word_vec_size)
        self.rnn = nn.LSTM(
            input_size=word_vec_size,
            hidden_size=hidden_size,
            num_layers=n_layers,
            dropout=dropout_p,
            batch_first=True,
            bidirectional=True,
        )
        self.generator = nn.Linear(hidden_size * 2, n_classes)
        # We use LogSoftmax + NLLLoss instead of Softmax + CrossEntropy
        self.activation = nn.LogSoftmax(dim=-1)

- Embedding Layer -> LSTM(bidirectional=True) -> Linear Layer -> LogSoftmax
